website/docs/installation.mdx

```diff
@@ -564,3 +564,23 @@ The quickest way to access your dashboard is by setting up a port forward:
 kubectl port-forward svc/ww-gitops-weave-gitops -n flux-system 9001:9001
 ```
 Then, [open the dashboard](http://localhost:9001/).
+
+## Installing the TF-Controller
+
+The [TF-Controller](https://weaveworks.github.io/tf-controller/) is a controller for Flux to reconcile Terraform resources in a GitOps way. 
+
+With Flux and the TF-Controller, Weave GitOps Enterprise makes it easy to add Terraform templates to clusters and continuously reconcile any changes made to the Terraform source manifest. 
+
+Check out our guide on how to [use Terraform templates](using-terraform-templates.mdx#), and why not try your hands at using it with the RDS example!
```

Comment:
extra `#` here `[use Terraform templates](using-terraform-templates.mdx#)` probably not required

Reply:
woops. good spot! fixed it, and verified that it leads to the guide :)